src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.GetProcessMemoryInfo.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    [StructLayout(LayoutKind.Sequential)]
+    internal struct ProcessMemoryCounters
+    {
+        public uint cb;
+        public uint PageFaultCount;
+        public ulong PeakWorkingSetSize;
+        public ulong WorkingSetSize;
+        public ulong QuotaPeakPagedPoolUsage;
+        public ulong QuotaPagedPoolUsage;
+        public ulong QuotaPeakNonPagedPoolUsage;
+        public ulong QuotaNonPagedPoolUsage;
+        public ulong PagefileUsage;
+        public ulong PeakPagefileUsage;
+    }
+
+    internal partial class Kernel32
+    {
+        [DllImport(Libraries.Kernel32, SetLastError = true)]
+        internal static extern bool K32GetProcessMemoryInfo(IntPtr handleProcess, out ProcessMemoryCounters pmCounter, uint cb);
```

Comment:
`pmCounter` should be `ref` rather than `out`.

Reply:
The arguments here should be named `Process`, `ppsmemCounters`, and `cb`, to match the native declaration.